packages/data-broker/src/__tests__/DataBroker/DataBroker.test.js

```diff
@@ -4,26 +4,30 @@
  */
 
 import { DataBroker } from '../../DataBroker';
-import * as GetModels from '../../getModels';
 import { DATA_BY_SERVICE, DATA_ELEMENTS, DATA_GROUPS } from './DataBroker.fixtures';
 import { stubCreateService, createModelsStub, createServiceStub } from './DataBroker.stubs';
 
+const dataSources = Object.values(DATA_ELEMENTS).concat(Object.values(DATA_GROUPS));
+const mockModels = createModelsStub(dataSources);
```

Comment:
This variable must start with `mock`, since it's used inside `jest.fn().mockImplementation` below. From the [jest docs](https://jestjs.io/docs/en/es6-class-mocks#calling-jestmockdocsenjest-objectjestmockmodulename-factory-options-with-the-module-factory-parameter):

> A limitation with the factory parameter is that, since calls to jest.mock() are hoisted to the top of the file, it's not possible to first define a variable and then use it in the factory. An exception is made for variables that start with the word 'mock'.